static/global.css

```diff
@@ -13,3 +13,41 @@ body {
 	width: calc(100vw - 240px);
 	margin: 0 auto;
 }
+
+/********* Navbar *********/
+.navbar {
+	display: flex;
+	align-items: center;
+	justify-content: space-evenly;
+	padding: 10px 20px;
+	background-color: #f5f5f5;
+	color: #1a110f;
+}
+
+.logo {
+	display: flex;
+	align-items: center;
+}
+
+.logo img {
+	height: 40px;
+}
+
+.navbar-tabs {
+	list-style: none;
+	display: flex;
+	align-items: center;
+}
+
+.navbar-tabs li {
+	margin-right: 5vh;
+}
+
+.navbar-tabs a {
+	color: rgb(26, 17, 15, 0.5);
+	text-decoration: none;
+}
+
+.connexionButton {
+	color: #f5f5f5 !important;
```

Comment:
Do not use `important`. See if `f5f5f5` corresponds to a SASS variable that we know.

Reply:
Ok for this one ✅